functional/main_test.py

```diff
@@ -0,0 +1,49 @@
+import os
+import unittest
+
+import chromedriver_autoinstaller
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from pyvirtualdisplay import Display
+
+class MainPageTests(unittest.TestCase):
+    def setUp(self):
+        display = Display(visible=0, size=(800, 800))
+        display.start()
+        chromedriver_autoinstaller.install()
+        self.BASE_URL = os.getenv("BASE_URL")
+        self.driver = webdriver.Chrome()
+
+    def tearDown(self):
+        self.driver.close()
+
+    def test_main(self):
+        self.driver.get(self.BASE_URL)
+        assert self.driver.title == 'Paint Drying'
+
+    def test_register_link(self):
+        self.driver.get(self.BASE_URL)
+        assert self.driver.title == 'Paint Drying'
```

Comment:
You are already asserting that `title == 'Paint Drying'` in test_main(). It's not necessary to check this in every single test.